Apps/Immunization/test/unit/Delegates.Test/ImmunizationDelegateTests.cs

```diff
@@ -145,9 +143,8 @@ public async Task GetImmunizationsThrowsException()
             PhsaResult<ImmunizationResponse> phsaResponse = new()
             {
                 Result = new ImmunizationResponse(
-                    new List<ImmunizationViewResponse>
-                        { expectedViewResponse },
-                    new List<ImmunizationRecommendationResponse>()),
+                    [expectedViewResponse],
+                    []),
```

Comment:
Nit: now that these lines are much shorter, they can probably all fit on a single line.

Reply:
Need to configure Rider.  Change next time.